### ISSUE TITLE
Case-Insensitive Configuration Keys

### ISSUE THREAD

**Issue by rawkode** [GT8Online/weave#11]
We should probably lower-case all configuration keys before applying `handle_configuration`.

**Comment by rawkode** [comment]
Why?

1. It would allow people to move to different loaders without having to modify case (When environment "standard" would be uppercase, but file is lower)
2. Should we really allow these to be handled differently?

```elixir
def handle_configuration(key = "API_KEY", key)
def handle_configuration(key = "api_key", key)
```

**Comment by rawkode** [comment]
Closed by #13
